config/application.yml.example

```diff
@@ -98,7 +98,9 @@ development:
   otp_valid_for: '5'
   password_pepper: 'f22d4b2cafac9066fe2f4416f5b7a32c6942d82f7e00740c7594a095fa8de8db17c05314be7b18a5d6dd5683e73eadf6cc95aa633e5ad9a701edb95192a6a105'
   password_strength_enabled: 'true'
-  proofing_vendors: 'mock'
+  proofing_vendor_finance: 'mock'
+  proofing_vendor_phone: 'mock'
+  proofing_vendor_profile: 'mock'
```

Comment:
What do you think about making the suffix the prefix: `finance_proofing_vendor`? That way it describes the kind of proofing the vendor does, as opposed to sounding like something the vendor has. `proofing_vendor_profile` sounds like the profile of the proofing vendor.

Reply:
That works for me. I used `proofing_vendor_*` because then they're grouped together in an alphabetized list.